### DISCUSSION TITLE
Order content by taxonomy value [backend]

### DISCUSSION THREAD

**Discussion by peterboorsma** [bolt/core#1319] [Ideas / Feature requests]
Bolt 4 does not seem to order content by its taxonomy values.
One reason to use a 'grouping' taxonomy for a contenttype is to get some order in the backend listing of a contenttype. Like this:
```
The main group
- a page
- another page

The other stuff
- yet another page
- and another
```

Am I missing a setting for this? Or is it not implemented (yet)?
I vaguely remember this was quite a challenge to implement in Bolt 3. Maybe that is the reason why this is not a feature (yet)?
A simple useful solution (for now) is to add a filter for this in de sidebar. 

Bolt 4.0.0 RC16

**Comment by bobdenotter** [comment]
Yeah, I liked that feature too, and I really want to bring it back. But, as you know the implementation in v3 was flawed. 

I'd like to bring it back, but then make sure it works as expected. Something for 4.1, though! :-)
